app/views/explore/index.html.haml

```diff
@@ -1,18 +1,5 @@
 - content_for :after_title, " - #{t(Features.wiki_ed? ? 'application.explore' : 'courses_generic.explore')}"
-
-%header.main-page
-  .container
-    %h1=  ENV['dashboard_title']
-
+:javascript
+  Features.course_string_prefix = "#{@presenter.course_string_prefix}"
```

Comment:
I guess this is the only dynamic part of the explore page now? It looks like initializing a CoursesPresenter is going to also do a query for the campaign courses, even if it's not going to need it (as with serving this HTML route), so it might make sense to just use `Features.default_course_string_prefix` instead of the presenter. I think in this context it will be the same.

Reply:
I thought the value might be different for different campaigns. I suppose that case might not be relevant for these templates.